### ISSUE TITLE
Twig Runtime Error on Regular Segments page

### ISSUE THREAD

**Issue by Rommert** [putyourlightson/craft-campaign#294]
### Describe the bug
I am curently testing some mail segments for a client. When creating a new Regular Segment, I get an 'Twig Runtime Error'. 

 in /app/vendor/putyourlightson/craft-campaign/src/templates/segments/_includes/condition.html

```Key "handle" for array with keys "type, column, name, options" does not exist.```

### User fields
- 20 fields attached to a user/contact 
-checkboxes, radiobuttons, text and textarea's, nothing special

I guess something is going wrong with the radiobutton fields. Can you help me out?

### Versions
- Craft version: 3.7.33
![Schermafbeelding 2022-02-23 om 10 58 04](https://user-images.githubusercontent.com/2290530/155296999-294f2c8c-46f8-4cc5-a723-a5f99aadca94.png)

**Comment by bencroker** [comment]
Can you show me the condition logic that is failing in Twig?

**Comment by Rommert** [comment]
Hi @bencroker, off course!

File: vendor/putyourlightson/craft-campaign/src/templates/segments/_includes/condition.html



```    {% for field in availableFields %}
            {% if field.options %}
                <div class="conditionValue field-{{ field.type|replace({'\\': '-'}) }}-{{ field.handle }} hidden">
                    <div class="select">
                        <select name="{{ fieldName }}" disabled>
                            {% for option in field.options %}
                                <option value="{{ option.value }}" {{ value == option.value ? 'selected' }}>{{ option.label }}</option>
                            {% endfor %}
                        </select>
                    </div>
                </div>
            {% endif %}
        {% endfor %}
```
Reproduce screenrecording: https://www.dropbox.com/s/62k66oa8lie6moy/Schermopname%202022-02-23%20om%2014.48.02.mov?dl=0

**Comment by bencroker** [comment]
Replace `{{ field.handle }}` with `{{ fieldName }}`?

**Comment by Rommert** [comment]
Yes, that did the trick! Merci! I will create a pull request for this one.

**Comment by bencroker** [comment]
Thanks, fixed in https://github.com/putyourlightson/craft-campaign/commit/6cb9de50abda84bfd288e397e992bb835226828c and released in version 1.22.4.
